src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -52,6 +52,28 @@
    :render
    (fn [{:keys [state props]}]
      [:div {}
+      [comps/ModalDialog
+       {:height 250
+        :width 400
+        :content [:div {}
+                  [:div {:style {:backgroundColor "#fff"
+                                 :borderBottom (str "1px solid " (:line-gray style/colors))
+                                 :padding "20px 48px 18px"
+                                 :fontSize "137%" :fontWeight 400 :lineHeight 1}}
+                   "Import a Method Configuration"]
+                  [:div {:style {:padding "22px 48px 40px" :backgroundColor (:background-gray style/colors)}}
+                   "Destination Name : "
+                   (style/create-text-field  {:style { :ref "destinationName" }} )
+                   [:br {} ]
+                   "Destination Namespace : "
+                   (style/create-text-field  {:style { :ref "destinationNamespace" }} )
+                   [:br {} ]
+                   [comps/Button
+                    {
+                     :icon :plus
+                     :onClick (fn [] (swap! state assoc :import-mc-modal-shown false))}]]]
```

Comment:
In Clojure, booleans end with a '?'. Also, this name suggests there will be multiple modals (otherwise, the name would be `:modal-shown?` rather than `:import-method-configuration-modal-shown?` or the shortened version you have. Is that true? I suggest you rename this to `:show-modal?`. Avoid the past tense for consistency ("shown" -> "show").